src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Tabs.js

```diff
@@ -1,5 +1,6 @@
 export class Tabs {
   initEventListeners () {
+    this.loadTab()
```

Comment:
This shouldn't be in `initEventListeners()`, but should be triggered here: https://github.com/sumocoders/Framework/blob/master/src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Index.js#L39

Reply:
Fixed 9600259